File: Beginner/C#/1013 - The Greatest.cs
using System;

class Program
{
    static void Main(string[] args)
    {

        //Declare Variables
        string values = Console.ReadLine();
        string[] values2 = values.Split(' ');
        int A = Convert.ToInt32(values2[0]);
        int B = Convert.ToInt32(values2[1]);
        int C = Convert.ToInt32(values2[2]);

        //Calculate
        int majorAB = (A + B + Math.Abs(A - B)) / 2;
        int majorABC = (majorAB + C + Math.Abs(majorAB - C)) / 2;

        //Print
        Console.WriteLine("{0} eh o maior", majorABC);

    }
}

File: Beginner/Python 3/1013 - The Greatest.py
# Declare Variables
numbers = input().split(' ')
a = int(numbers[0])
b = int(numbers[1])
c = int(numbers[2])

# Calculate
preGreater = (a + b + abs(a - b)) / 2
greater = (preGreater + c + abs(preGreater - c)) / 2

# Show result
print("{} eh o maior".format(int(greater)))
